### PR TITLE
[3750][1617] added range property to password form

### PR DESCRIPTION
**Pull request by pablobentivengo**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/3750
* Fixes https://github.com/scandipwa/scandipwa/issues/1617

**Problem:**
* No password length validation

**In this PR:**
* Added range property to password forms with values: min: 8 and max: 64

### CODE REVIEW

**Comment by Elena-Evsa** [comment]
Issue #1617 - failed - still can reproduce in every 'new' password form

Comments from pr #3924: 
Issue 1 - passed 
Issue 2 - passed
Issue 3 - Failed

Comments from pr #4088
Issue 4 - Failed

File: packages/scandipwa/src/component/MyAccountCustomerForm/MyAccountCustomerForm.component.js [hunk 1]
```diff
@@ -31,7 +31,8 @@ export class MyAccountCustomerForm extends FieldForm {
         handlePasswordInput: PropTypes.func.isRequired,
         email: PropTypes.string,
         currentPassword: PropTypes.string,
-        vatNumberRequired: PropTypes.bool.isRequired
+        vatNumberRequired: PropTypes.bool.isRequired,
+        range: PropTypes.objectOf(PropTypes.number).isRequired
```
Comment (ejnshtein) [inline packages/scandipwa/src/component/MyAccountCustomerForm/MyAccountCustomerForm.component.js:35]: Maybe using type `PropTypes.shape({ min: PropTypes.number, max: PropTypes.number }).isRequired` will be more appropriate here?

File: packages/scandipwa/src/component/MyAccountSignIn/MyAccountSignIn.component.js [hunk 2]
```diff
@@ -29,7 +29,8 @@ export class MyAccountSignIn extends PureComponent {
         handleCreateAccount: PropTypes.func.isRequired,
         isCheckout: PropTypes.bool.isRequired,
         state: SignInStateType.isRequired,
-        emailValue: PropTypes.string.isRequired
+        emailValue: PropTypes.string.isRequired,
+        range: PropTypes.objectOf(PropTypes.number).isRequired
```
Comment (ejnshtein) [inline packages/scandipwa/src/component/MyAccountSignIn/MyAccountSignIn.component.js:33]: Same here.

**Review by ejnshtein — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/MyAccountSignIn/MyAccountSignIn.component.js (hunk 2 above)
Comment (pablobentivengo) [reply packages/scandipwa/src/component/MyAccountSignIn/MyAccountSignIn.component.js]: Done

File: packages/scandipwa/src/component/MyAccountCustomerForm/MyAccountCustomerForm.component.js (hunk 1 above)
Comment (pablobentivengo) [reply packages/scandipwa/src/component/MyAccountCustomerForm/MyAccountCustomerForm.component.js]: Done

**Comment by carinadues** [comment]
Issue 1: Enter a password with only one class of characters.
The error message appears, but the borders of fields for entering password stay green after clicking on save without the right minimum number os types of characters.
image

Issue 2: Please add the same logic to the New password on page "Change my password" (the user can reach it after the 'forgot password' procedure)
- Red with text if less than the minimum number of characters allowed
- Red with text if they don't match
- Fields borders become red if they don't enter the minimum number of allowed types of characters

image

**Comment by Elena-Evsa** [comment]
**Issue #1617** - still can reproduce in every 'new' password form - Failed
length of password depends on 'Minimum Password Length' (now = 7), but on FE still 8 characters
image

**Comments from pr #3924:** 
Issue 1 - passed 
Issue 2 - passed
Issue 3 - passed

**Comments from pr #4088**
Issue 4 - passed

**Comment by aleksandrakorolova** [comment]
1. 'Minimum {number} characters!' is displayed for confirm password field in my account
Screenshot 2022-02-07 at 12 57 07
**Expected result:**
This message is displayed only below New password form as it works in create a new account form 

2. Issue 1617 - still, can reproduce
Screenshot 2022-02-07 at 13 19 38

**Review by ejnshtein — APPROVED** [review]

**Pull request merged**